sim/src/mechanics/car.rs

```diff
@@ -214,12 +214,12 @@ impl Car {
                 CarState::Queued { .. } => CarStatus::Moving,
                 CarState::WaitingToAdvance { .. } => CarStatus::Moving,
                 CarState::Crossing(_, _) => CarStatus::Moving,
-                // Eh they're technically moving, but this is a bit easier to spot
-                CarState::Unparking(_, _, _) => CarStatus::Parked,
-                CarState::Parking(_, _, _) => CarStatus::Parked,
+                CarState::Unparking(_, _, _) => CarStatus::Moving,
```

Comment:
We used to style both the Parking and Unparking car as parked (grayed out). Now that we're showing the parking intent, I think it makes sense to color the car until it's full parked.

However, is_parking doesn't cover the "unparking" case, so we don't show the intent for unparking. 

Do you think it would be too confusing to show the "parking" icon for people who are unparking? I think it would be unlikely to confuse people, but 🤷 

I guess in that case I'd also rename `DrawCarInput.is_parking` to something like `DrawCarInput.show_parking_intent`.

Reply:
example of showing indicator while unparking

![unparking mov](https://user-images.githubusercontent.com/217057/95277718-52158c00-0803-11eb-8613-93121ff6de01.gif)

Reply:
> Do you think it would be too confusing to show the "parking" icon for people who are unparking?

Nah, seems clear enough, and it's consistent with sliding into the parking spot. I'd vote do it.

Aside: Is "unparking" an awful silly term -- was it clear what it meant when you first saw it; is there anything better?

Reply:
I admit it feels silly, but I don't know of a better alternative. Apparently it's not a concept I often articulate. I'd say keep it!